CODE_OF_CONDUCT.md

```diff
@@ -15,7 +15,7 @@ Examples of behavior that contributes to creating a positive environment
 include:
 
 * Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
+* Being respectful towards different viewpoints and experiences
```

Comment:
This change is not needed, "respectful of" is perfectly valid English. Please change it back.
```suggestion
* Being respectful of differing viewpoints and experiences
```